purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order.py

```diff
@@ -21,10 +21,18 @@ class PurchaseRequestLineMakePurchaseOrder(models.TransientModel):
     def default_get(self, fields):
         res = super(PurchaseRequestLineMakePurchaseOrder, self).\
             default_get(fields)
-        request_line_obj = self.env['purchase.request.line']
```

Comment:
Ok this variable may only be used once but as long as the purpose of the pull request is different I would prefer to skip it for now, the relevant change is the if statement

Reply:
OK. Do you want me to make another PR without it?

Reply:
You can do the change in this PR but I will not block the PR for this. It is just for reviewers to focus on what the PR is about.